core/arch/arm/include/mm/core_mmu.h

```diff
@@ -111,11 +111,14 @@ struct core_mmu_phys_mem {
 	size_t size;
 };
 
-#define register_phys_mem(type, addr, size) \
-	static const struct core_mmu_phys_mem __phys_mem_ ## addr \
+#define __phys_mem2(type, addr, size, id) \
```

Comment:
Not that it matters much, but these two new `__phys_mem1` and `__phys_mem2` could have been made to cover what `__register_sdp_mem1` and `__register_sdp_mem2` is doing too.